crates/storage/src/schema/revision_0030.rs

```diff
@@ -0,0 +1,19 @@
+use anyhow::Context;
+
+/// This migration creates the table for the class commitment tree.
+///
+/// Normally this is not necessary (because opening the tree will automatically
+/// create the tables), but for tests it's useful because otherwise the Python
+/// subprocess will just throw errors.
```

Comment:
Yes, if anything we should really be adding the other tables in a migration as well.

Reply:
I've added the other two tables as well to this migration. The test fixture DB is not affected though since that already has those (but was missing `tree_class`).